src/components/panels/NewsPanel.vue

```diff
@@ -0,0 +1,49 @@
+<template>
+  <b-col md="6">
+        <h3 class="text-center">{{headertext}}</h3>
+        <b-carousel id="infoscreen-carousel" :interval="9000" :indicators="false" :controls="true">
+          <b-carousel-slide   v-for="(newsitem,index) in news" :key="index"
+          v-if="newsitem.translations['en']"
+          :text="newsitem.translations['en']">
+          <img slot="img" class="d-block   w-100" width="1024" height="250"
+          :src="newsitem.imageLink" alt="image slot">
+          <p style="color: 'black'!important">
+            <b>Posted: {{newsitem.eta}} | More Info: <a :href="newsitem.link" target="_blank">Click</a>
```

Comment:
need to use a fallback for the newsitem link.
![image](https://user-images.githubusercontent.com/7128721/47460445-2464e300-d7a4-11e8-8266-07300b249b34.png)

Reply:
sugest one,  bootstrap doesn't  allow  text  only  slides in caroussel

Reply:
https://i.imgur.com/CNrsc7V.png